lua/lazyvim/plugins/extras/coding/tabnine.lua

```diff
@@ -8,14 +11,14 @@ return {
         "tzachar/cmp-tabnine",
         build = "./install.sh",
         dependencies = "hrsh7th/nvim-cmp",
-        -- Only limit top 3 suggestions from Tabnine
-        config = function()
+        opts = {
+          max_lines = 1000,
+          max_num_results = 3,
+          sort = true,
+        },
+        config = function(_, opts)
```

Comment:
Is there a reason for the `config` function, since you don't seem to alter the default `lazy.nvim` config function behavior? I believe this is redundant? Pls correct me if I'm wrong.